.github/workflows/ci.yaml

```diff
@@ -584,6 +615,13 @@ jobs:
         if: ${{ matrix.container-runtime == 'docker' }}
         run: |
           cargo test --target=x86_64-unknown-linux-gnu -p mirrord-tests --no-default-features --features cli -- --test-threads=6
+      - uses: metalbear-co/setup-protoc@3ea1d70ac22caff0b66ed6cb37d5b7aadebd4623
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - run: cargo build --manifest-path=./Cargo.toml -p mirrord --features wizard
+      - run: |
+          ./target/debug/mirrord wizard &
```

Comment:
Wonder how flaky this could be hmm.

We could check for some progress message in stdout before `curl`ing.

I'm ok with not changing this, but maybe add a comment here with ideas if this starts failing on people.